### ISSUE TITLE
Allow users to specify the line strokes / colors

### ISSUE THREAD

**Issue by twni2016** [aimhubio/aim#2295]
## 🚀 Feature

Allow users to change the colors and strokes in the charts with their preference (not by shuffling). In the table of runs, users can click the colors / strokes in the group column for specification. 

### Motivation

In the line charts, users can only change the colors / strokes of curves by clicking the `shuffle` button. But sometimes users have their preferences on the styles. For example, I'd like to assign the strokes into solid `―` and dashed `--` ones, for two groups of curves. However, it is hard for me to keep shuffling the strokes to get the desired pairs.

**Comment by gorarakelyan** [comment]
@twni2016 thanks for the suggestion! 
What do you think the expected behavior should be when updating the groups? 
So for example metrics were divided into 3 groups - g0, g1, g2 based on hparams.learning_rate. What should happen when dividing the same set of metrics into 4 groups based on batch_size? Should the applied styles be reset?

**Comment by twni2016** [comment]
@gorarakelyan I think it's better to keep the existing styles instead of resetting, to reduce the cognitive load of users caused by context switching.
